CONTRIBUTING.md

```diff
@@ -224,65 +239,86 @@ but do not include `Documentation/Metrics/allMetrics.yaml` in your PR
 
-Optimizations and limit of architecture (list of possible CPU instructions) are set using this `cmake` option
-in addition to the other options:
+Check the [`.circleci/base_config.yml`](.circleci/base_config.yml) file for the
+default `build-docker-image` value to ensure it's the matching image tag for the
+version you want to build.
 
-```
--DTARGET_ARCHITECTURE
-```
+You can start the build container as follows:
 
-Oskar uses predefined architecture which is defined in `./VERSIONS` file or `sandybridge` if it's not defined.
+    cd arangodb
+    docker run -it -v $(pwd):/root/project arangodb/ubuntubuildarangodb-devel:3
 
-Note: if you use more modern architecture for optimizations or any additional implementation with extended
-set of CPU instructions please notice that result could be different to the default one.
+In the fish shell of the container, run the following commands for a release-like
+build using the officially supported Clang compiler (also see the [`VERSIONS`](VERSIONS) file):
 
-For building the ArangoDB starter checkout the
-[ArangoDB Starter](https://github.com/arangodb-helper/arangodb).
+    cd /root/project
+    cmake --preset community -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=lld" -DCMAKE_LIBRARY_PATH=$OPENSSL_ROOT_DIR/lib -DOPENSSL_ROOT_DIR=/opt
+    cmake --build --preset community --parallel (nproc)
 
-### Building the Web Interface
+For a debug build, you can use the `community-developer` preset instead.
 
-To build web interface, also known as the Web UI, frontend, or _Aardvark_, [Node.js](https://nodejs.org/)
-and [Yarn](https://yarnpkg.com/) need to be installed. You can then use CMake in the
-build directory:
+The default target CPU architecture is `sandybridge` for the x86-64 platform and
+`graviton1` for 64-bit ARM chips. You can specify a different architecture to
+enable more optimizations or to limit the compiler to the available instructions
+for a particular architecture:
 
-    cmake --build . --target frontend
+    cmake --preset community -DTARGET_ARCHITECTURE=native ...
 
-Note that as of 3.12 the frontend target is part of the default target. To avoid building the frontend and
-skip the CMake checks for yarn and node you can disable this using `-DUSE_FRONTEND=Off`.
+On ARM, disable the `USE_LIBUNWIND` option:
 
-For Oskar you may use the following:
+    cmake --preset community -DUSE_LIBUNWIND=Off ...
 
-    shellInAlpineContainer
+To build specific targets instead of all, you can specify them like shown below:
 
-    apk add --no-cache nodejs yarn && cd /work/ArangoDB/build && cmake --build . --target frontend
+    cmake --build --preset community --target arangod arangosh arangodump arangorestore arangoimport arangoexport arangobench arangovpack frontend
 
-To remove all available node modules and start a clean installation run:
+You can find the output files at `build-presets/<preset-name>/`,
+with the compiled executables in the `bin/` subfolder.
 
```

Comment:
To collect the files and bring them into a release-like structure:
`cmake --install ./build-presets/community --prefix package`
(no --preset support, --prefix is the target folder, may not work if not all targets were built)